airbyte-webapp/src/pages/connections/ConnectionSettingsPage/ConnectionSettingsPage.tsx

```diff
@@ -32,3 +34,10 @@ export const ConnectionSettingsTab: React.FC = () => {
     </div>
   );
 };
+
+export const ConnectionSettingsPage: React.FC = () => {
+  const { connection } = useConnectionEditService();
+  const isConnectionDeleted = connection.status === ConnectionStatus.deprecated;
+
+  return isConnectionDeleted ? <Navigate replace to=".." /> : <ConnectionSettingsPageInner />;
```

Comment:
nitpick: could we navigate to an explicit route here? I find it easier to understand than "back one page" (`to=".."`), which requires you to go look at the `<Routes>` and figure out what that means in the current context. I think it also makes editing routes later easier, because we can see all instances of navigation to a particular route.

Reply:
I wish we had a better way to generate paths that don't involve a lot of reconstruction. In this case, we'd have to pull the workspaceId to navigate to the right place.

Reply:
Agreed, however I would still prefer explicit routes to a relative one. Getting the workspace ID Is pretty straightforward with `useCurrentWorkspaceId()`. I don't feel super strongly though, so feel free to keep it this way if you like.

Reply:
Maybe a comment on where the user is supposed to end up?